README.md

```diff
@@ -11,26 +11,46 @@ as it internally uses [BuildKit](https://github.com/moby/buildkit)'s DAG solver.
 have to do is replace `docker` with `img` in your scripts, command line, and/or life.
 
-You might also be interested in reading: 
-* [the original design doc](https://docs.google.com/document/d/1rT2GUSqDGcI2e6fD5nef7amkW0VFggwhlljrKQPTn0s/edit?usp=sharing)
-* [a blog post on building images securely in Kubernetes](https://blog.jessfraz.com/post/building-container-images-securely-on-kubernetes/)
+## Goals
+
+This a glorified cli tool built on top of
+[buildkit](https://github.com/moby/buildkit). The goal of this project is to be
+able to build container images as an unprivileged user.
 
-Currently you can run it as unprivileged using `native` backend.
-On some distros such as Ubuntu, `overlayfs` backend might work as well.
-You can learn more about the snapshotter backends [here](#snapshotter-backends).
+Running unprivileged allows companies who use LDAP and other login mechanisms
+to use `img` without needed root. This is very important in HPC environments
+and academia as well.
 
-However, it does not currently work as an unprivileged user _in_ a container
-due to the fact that it cant mount `proc` inside the container. See
-[opencontainers/runc#1658](https://github.com/opencontainers/runc/issues/1658)
-for more info.
+Currently this works out of the box on a Linux machine if you install via 
+the directions covered in [installing from binaries](#binaries). This
+installation will ensure you have the correct version of `img` and also the
+patched version of `runc`.
 
-But it will work as an unprivileged user on your host with a `runc` that is
-compiled from the following branch: [AkihiroSuda/runc/tree/demo-rootless](https://github.com/AkihiroSuda/runc/tree/demo-rootless). I also uploaded one here: [https://misc.j3ss.co/tmp/runc](https://misc.j3ss.co/tmp/runc), if you trust me.
+The ultimate goal is to also have this work inside a container. There are
+patches being made to container runtimes and Kubernetes to make this possible. 
+For the on-going work toward getting patches into container runtimes and
+Kubernetes, see:
 
-(For the on-going work toward integrating runc with these patches to `buildkit`, please refer to [moby/buildkit#252](https://github.com/moby/buildkit/issues/252#issuecomment-359696630) and [AkihiroSuda/buildkit_poc@511c7e71](https://github.com/AkihiroSuda/buildkit_poc/commit/511c7e71156fb349dca52475d8c0dc0946159b7b).)
+- [moby/moby#36644](https://github.com/moby/moby/pull/36644)
+- [kubernetes/community#1934](https://github.com/kubernetes/community/pull/1934)
+- [kubernetes/kubernetes#64283](https://github.com/kubernetes/kubernetes/pull/64283)
 
+For the on-going work on `runc` patches upstream, please see:
 
-**TLDR;** will work unprivileged on your host not in a container til we work out some kinks.
+- [opencontainers/runc#1692](https://github.com/opencontainers/runc/pull/1692)
+- [opencontainers/runc#1759](https://github.com/opencontainers/runc/pull/1759)
```

Comment:
This one got merged just now (thanks @hqhq !)

Reply:
fixed the image size and updated, ill update the fork :)